### DISCUSSION TITLE
ppocr ERROR: rec_algorithm must in ['CRNN', 'SVTR_LCNet']

### DISCUSSION THREAD

**Discussion by fang-water** [PaddlePaddle/PaddleOCR#12307] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

**Comment by Sunting78** [comment]
您好，运行命令提供一下。
